### PR TITLE
add IPv6 support to the e2e PreStop test

### PR DESCRIPTION
**Pull request by aojea**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation

/kind failing-test

> /kind feature
> /kind flake

**What this PR does / why we need it**:

It adds IPv6 support to the e2e test 
`[k8s.io] [sig-node] PreStop should call prestop when killing a pod [Conformance]`

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

xref: #70248

https://testgrid.k8s.io/conformance-kind#kind%20(IPv6),%20master%20(dev)

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by BenTheElder** [comment]
/unassign @listx 

doesn't look like this needs any image changes :^)

**Review by BenTheElder — APPROVED** [review]
/lgtm
/approve
trivial correctness fix joining host + port

**Comment by BenTheElder** [comment]
/priority important-soon
/ok-to-test
^^ please join the org :-)

File: test/e2e/node/pre_stop.go [hunk 1]
```diff
@@ -78,6 +79,8 @@ func testPreStop(c clientset.Interface, ns string) {
 	podOut, err := c.CoreV1().Pods(ns).Get(podDescr.Name, metav1.GetOptions{})
 	framework.ExpectNoError(err, "getting pod info")
 
+	podUrl := net.JoinHostPort(podOut.Status.PodIP, "8080")
```
Comment (BenTheElder) [inline test/e2e/node/pre_stop.go]: golint is failing, this should be:
```suggestion
	podURL := net.JoinHostPort(podOut.Status.PodIP, "8080")
```

File: test/e2e/node/pre_stop.go [hunk 2]
```diff
@@ -92,7 +95,7 @@ func testPreStop(c clientset.Interface, ns string) {
 						PreStop: &v1.Handler{
 							Exec: &v1.ExecAction{
 								Command: []string{
-									"wget", "-O-", "--post-data=" + val, fmt.Sprintf("http://%s:8080/write", podOut.Status.PodIP),
+									"wget", "-O-", "--post-data=" + val, fmt.Sprintf("http://%s/write", podUrl),
```
Comment (BenTheElder) [inline test/e2e/node/pre_stop.go]:
```suggestion
									"wget", "-O-", "--post-data=" + val, fmt.Sprintf("http://%s/write", podURL),
```

**Comment by BenTheElder** [comment]
/lgtm cancel
(to stop the bot from spamming /retest until the golint issue is fixed)

File: test/e2e/node/pre_stop.go (hunk 2 above)
Comment (aojea) [reply test/e2e/node/pre_stop.go]: I amended the commit with the changes to not mess the git history

File: test/e2e/node/pre_stop.go (hunk 1 above)
Comment (aojea) [reply test/e2e/node/pre_stop.go]: amended your suggestion to the current commit

**Comment by aojea** [comment]
@BenTheElder checks green now

**Review by BenTheElder — APPROVED** [review]
/lgtm

**Pull request merged**